ChromeExtension/popup.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>  
+<html>
+    <head>
+        <style>
+            button {height: 30px;width: 30px;outline: none;}      

```

Comment:
Just a side note: this is fine if you don't plan on adding much more css. But for more complex pages, I'd recommend creating a separate css file